### PR TITLE
Adding footer for urban flows

### PR DESCRIPTION
**Pull request by dambem**
## Description of this PR
Adding a footer that urban flows wanted to add. All simple stuff

## Checklist
- [ ] Tests all pass

### CODE REVIEW

File: ClimateMonitorV2/views/index.pug [hunk 1]
```diff
@@ -119,11 +119,14 @@ block content
...
   script(src="https://d3js.org/d3.v5.min.js")
   script(src="https://cdn.jsdelivr.net/npm/litepicker/dist/js/main.js")
 
-  footer
-    a(href="https://www.gov.uk/government/publications/health-matters-air-pollution/health-matters-air-pollution?fbclid=IwAR1WCj4WuVHtGuDehhsQZ682MUpF4MIdM2AL8i4Z6UvyjnIkOiePaYjrQks") Information and images from Gov.UK
+  footer(id = "sticky-footer" class="py-4 bg-light text-white-50")
+    div(class="container text-center")
+        a(href="https://urbanflows.ac.uk/")
+            img(class="img-fluid", src="images/urbanflows.png", href="https://urbanflows.ac.uk/") 
+        //a(href="https://www.gov.uk/government/publications/health-matters-air-pollution/health-matters-air-pollution?fbclid=IwAR1WCj4WuVHtGuDehhsQZ682MUpF4MIdM2AL8i4Z6UvyjnIkOiePaYjrQks") Information and images from Gov.UK
```
Comment (M-Davies) [inline ClimateMonitorV2/views/index.pug]: Can we remove the comment? It's on the about page so we shouldn't really need it again here

**Review by M-Davies — CHANGES_REQUESTED** [review]
One small nitpick

File: ClimateMonitorV2/views/index.pug (hunk 1 above)
Comment (dambem) [reply ClimateMonitorV2/views/index.pug]: Good point, removing it

**Review by M-Davies — APPROVED** [review]
Happy for this to be merged if you're done adding commits

**Pull request merged**